library/src/androidTest/java/org/xmtp/android/library/GroupTest.kt

```diff
@@ -84,8 +84,8 @@ class GroupTest {
             alixGroup.addMembers(listOf(caro.walletAddress))
             boGroup.sync()
         }
-        assertEquals(alixGroup.members().size, 3)
-        assertEquals(boGroup.members().size, 3)
+        assertEquals(alixGroup.members(alixClient).size, 3)
+        assertEquals(boGroup.members(boClient).size, 3)
```

Comment:
If your trying to get around the fact that members are now suspend just wrap all this stuff in a `runBlocking`

```suggestion
        runBlocking {
          assertEquals(alixGroup.members().size, 3)
          assertEquals(boGroup.members().size, 3)
        }
```